public/tracking.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<!-- Piwik -->
+		<script type="text/javascript">
+			var _paq = _paq || [];
+			_paq.push(['setDocumentTitle', document.domain + "/" + document.title]);
+			_paq.push(['trackPageView']);
+			_paq.push(['enableLinkTracking']);
+			(function() {
+				var u="https://stats.data.gouv.fr/";
+				_paq.push(['setTrackerUrl', u+'piwik.php']);
+				_paq.push(['setSiteId', 4]);
+				var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+				g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+			})();
+		</script>
+		<!-- End Piwik Code -->
+		<link rel="icon" href="favicon.ico">
+
+		<title>OpenFisca - Turn law into software</title>
+
+		<link rel="stylesheet" type="text/css" href="css/main.css">
+	</head>
+	<body>
+		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
+
+
+		<h1>Cookies and opt-out</h1>
+		<p>
+			When you visit this website, we leave a small text file (a "cookie") on your computer. This helps us measure how many visits we've had and what pages are most viewed.
+		</p>
+		<div id='optout'>
+			<iframe class="optout" src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=en"></iframe>
+		</div>
+		<h2>This website does not show a consent banner for cookies, why?</h2>
+		<p>
+			Our tracking service, <a href="https://www.piwik.org">Piwik</a> is configured in compliance with the <a href="https://www.cnil.fr/fr/solutions-pour-la-mesure-daudience">« Cookies » recommandations (French)</a> from the French authorities (<abbr title="Commission Nationale de l'Informatique et des Libertés">CNIL</abbr>)
```

Comment:
- English quotes are `“”`, not `« »`.
- Use [`hreflang`](https://www.w3.org/TR/html5/links.html#attr-hyperlink-hreflang) on the `a`.